e2e/src/runTest.ts

```diff
@@ -9,8 +9,19 @@ import { Client } from 'pg';
 // Expected parameter: path to the folder with the extension package.json
 async function main(): Promise<void> {
   try {
-    await prepareBigQuery();
-    await preparePostgres();
+    try {
```

Comment:
I think we don't need nested try/catch. You can just add log after prepareBigQuery:
```
await prepareBigQuery();
console.log('bigquery prepared successfully')
await preparePostgres();
```